index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="js/sounds.js"></script>
```

Comment:
if this JS file ends up being large we might want to move it to the bottom of the body--tags in the head are blocking

Reply:
Oh good point! I learned that in my class since that makes the HTML load first, and then JS has access to all HTML elements.